### ISSUE TITLE
Accessibility: Search in table toolbar and clickable table row aren't focusable by keyboard

### ISSUE THREAD

**Issue by DusanMilko** [IBM/carbon-components-angular#1510]
Accessibility concern in regards to table.

1. The magnify glass is not focusable by keyboard.

2. Tables can have a click listener on the row but aren't accessible by keyboard

Interact-able elements should be focusable by keyboard

<img width="686" alt="Screen Shot 2020-09-10 at 6 16 00 PM" src="https://user-images.githubusercontent.com/302239/92822669-056c9b80-f392-11ea-9a67-583cc317eda6.png">
